public/js/main.js

```diff
@@ -76,7 +76,9 @@ function getTargetFromQuery() {
 
 const connTarget = getTargetFromQuery();
 if (connTarget) {
-  startDebugging(connTarget, actions).then(() => {
+  startDebugging(connTarget, actions).then((tabs) => {
+    actions.newTabs(tabs);
```

Comment:
actually, we need `selectedTab` for sourcemaps so that we can get the tab url for source maps